### ISSUE TITLE
Arduino IDE support on Windows 10/11

### ISSUE THREAD

**Issue by askondro** [dineshannayya/riscduino_firmware#1]
On MSYS2 the compiler is already available.

Any plans on supporting Arduino IDE on Windows?

**Comment by dineshannayya** [comment]
Yes, I am in process of integrating the Support for Windows, Wait for 2 more weeks

**Comment by dineshannayya** [comment]
Now our board works with windows Arduino ide.
